src/transformers/models/detr/modeling_detr.py

```diff
@@ -1400,17 +1408,23 @@ def forward(
 
         >>> inputs = feature_extractor(images=image, return_tensors="pt")
         >>> outputs = model(**inputs)
-        >>> # model predicts bounding boxes and corresponding COCO classes
-        >>> bboxes, logits = outputs.pred_boxes, outputs.logits
 
-        >>> # get probability per object class and remove the no-object class
-        >>> probas_per_class = outputs.logits.softmax(-1)[:, :, :-1]
-        >>> objects_to_keep = probas_per_class.max(-1).values > 0.9
-
-        >>> ids, _ = probas_per_class.max(-1).indices[objects_to_keep].sort()
-        >>> labels = [model.config.id2label[id.item()] for id in ids]
-        >>> labels
-        ['cat', 'cat', 'couch', 'remote', 'remote']
+        >>> # convert outputs (bounding boxes and class logits) to COCO API
+        >>> target_sizes = torch.tensor([image.size[::-1]])
+        >>> results = feature_extractor.post_process(outputs, target_sizes=target_sizes)[0]
+
+        >>> for score, label, box in zip(results["scores"], results["labels"], results["boxes"]):
+        ...     box = [round(i, 2) for i in box.tolist()]
+        ...     # let's only keep detections with score > 0.9
+        ...     if score > 0.9:
+        ...         print(
+        ...             f"Detected {model.config.id2label[label.item()]} with confidence {round(score.item(), 3)} at location {box}"
```

Comment:
```suggestion
        ...             f"Detected {model.config.id2label[label.item()]} with confidence "
        ...             f"{round(score.item(), 3)} at location {box}."
```
Will be easier to read if the line is not too long.